lightning/src/ln/shutdown_tests.rs

```diff
@@ -48,7 +48,7 @@ fn pre_funding_lock_shutdown_test() {
 	mine_transaction(&nodes[0], &tx);
 	mine_transaction(&nodes[1], &tx);
 
-	nodes[0].node.close_channel(&OutPoint { txid: tx.txid(), index: 0 }.to_channel_id(), &nodes[1].node.get_our_node_id()).unwrap();
+	nodes[0].node.close_channel(&ChannelId::v1_from_funding_outpoint(OutPoint { txid: tx.txid(), index: 0 }), &nodes[1].node.get_our_node_id()).unwrap();
```

Comment:
How about keeping `OutPoint::to_channel_id()` and renaming to `to_v1_channel_id()`? I think it is logically OK, a bit more readable, and less change needed.

Reply:
Expanding what I mean:
- instead of `outpoint.to_channel_id()`
- keep `outpoint.to_v1_channel_id()`
- and not the `ChannelId::v1_from_funding_outpoint(outpoint)`

It's rather just syntax sugar